### ISSUE TITLE
Bug: ClinVar classification giving wrong call

### ISSUE THREAD

**Issue by jungkim2** [diskin-lab-chop/AutoGVP#76]
#### Provide the command used or report the bug here
ClinVar 1NR not picking most recent date as final call

example: 
ClinVar variation ID:327880
vcf_id: 18-9124950-C-T
Submission_summary: 
327880	Likely benign	Jan 17, 2019	-	Not Provided	CN517202:not provided	criteria provided, single submitter	clinical testing	germline:na	GeneDx	SCV001836824.1	NDUFV2	-
327880	Uncertain significance	Jan 13, 2018	This variant was observed in the ICSL laboratory as part of a predisposition screen in an ostensibly healthy population. It had not been previously curated by ICSL or reported in the Human Gene Mutation Database (HGMD: prior to June 1st, 2018), and was therefore a candidate for classification through an automated scoring system. Utilizing variant allele frequency, disease prevalence and penetrance estimates, and inheritance mode, an automated score was calculated to assess if this variant is too frequent to cause the disease. Based on the score, this variant could not be ruled out of causing disease and therefore its association with disease required further investigation. A literature search was performed for the gene, cDNA change, and amino acid change (if applicable). No publications were found based on this search. This variant was therefore classified as a variant of unknown significance for this disease.	OMIM:252010	CN257533:Mitochondrial complex I deficiency, nuclear type 1	criteria provided, single submitter	clinical testing	germline:na	Illumina Laboratory Services, Illumina	SCV000410202.3	NDUFV2	-
327880	Benign	Oct 17, 2022	-	MedGen:CN517202	CN517202:not provided	criteria provided, single submitter	clinical testing	germline:na	Invitae	SCV001056585.3	NDUFV2	-

With information above, most recent date is Oct 17,2022 we would expect final_call to be beign. 

Currently outputs below. 
vcf_id	Ref.Gene	Func.refGene	ExonicFunc.refGene	AAChange.refGene	Stars	Intervar_evidence	intervar_adjusted_call	ID	final_call	Reasoning_for_call
18-9124950-C-T	NA	NA	NA	NA	1NR	InterVar: Likely benign PVS1=0 PS=[0, 0, 0, 0, 0] PM=[0, 1, 0, 0, 0, 0, 0] PP=[0, 0, 0, 0, 0, 0] BA1=0 BS=[0, 0, 0, 0, 0] BP=[0, 0, 0, 1, 0, 0, 1, 0]	Not adjusted, clinVar	327880	Uncertain_significance	ClinVar

#### Add Session info
Run `sessionInfo()` and post the output below

**Comment by naqvia** [comment]
The piece of code that is causing this is: 
```
submission_summary_df <- vroom(input_submission_file, comment = "#",delim="\t", 
                              col_names = c("VariationID","ClinicalSignificance","DateLastEvaluated",       
                                           "Description","SubmittedPhenotypeInfo","ReportedPhenotypeInfo",   
                                           "ReviewStatus","CollectionMethod","OriginCounts","Submitter",
                                           "SCV","SubmittedGeneSymbol","ExplanationOfInterpretation"),
                              show_col_types = FALSE) %>% dplyr::select("VariationID","ClinicalSignificance") %>% 
                              group_by(VariationID) %>%
                              #arrange(DateLastEvaluated) %>%
                              dplyr::slice(1) %>%
                              ungroup()
```

According to our logic we choose the latest date, since there are likely multiple entries, however, this sorting did not work bc the date format in the submission files. We can possibly re-format date (year/month). Open to suggestions.

**Comment by naqvia** [comment]
This issue has been resolved. We now sort by date. Unfortunately, after some time we realized that the submission file itself had problems (they had GRCh37 when it was supposed to be only GRCh38 assembly). For that one variant that issue arises but we don't think its a general thing that occurs, and think its beyond the scope of this project to correct clinVar variant files. I think a note in the readme would suffice.
